testplan/runnable/interactive/base.py

```diff
@@ -513,6 +513,9 @@ def stop_test_resources(self, test_uid, await_results=True):
         if not await_results:
             return self._run_async(self.stop_test_resources, test_uid)
 
+        if self.report[test_uid].env_status == entity.ResourceStatus.STOPPED:
+            return
```

Comment:
This is a performance boost in general, if something is stopped we simply skip the entire process. Not sure if there is a way to trigger another stop of the environment resources than usual (in which case the button status protects us), but just to be sure.

Reply:
It's better that we check env_status not in [STOPPING, STOPPED]